src/App.jsx

```diff
@@ -5,31 +5,34 @@ import './App.css'
 
 function App() {
   const [count, setCount] = useState(0)
+  const [num, setNum] = useState(0);
 
+  function randomNumberInRange(min, max) {
+    // 👇️ Get the number between min (inclusive) and max (inclusive)
+    return Math.floor(Math.random() * (max - min + 1)) + min;
+  }
+
+  const handleClick = () => {
+    setNum(randomNumberInRange(1, 5));
+  };
+  const [num2, setNum2] = useState(0);
```

Comment:
```suggestion
  const [num2, setNum2] = useState(0);
  const handleClick = () => {
    setNum(randomNumberInRange(1, 5));
    setNum2(randomNumberInRange(1, 5));
  };
```
Du kannst hier gleich beide Zahlen auf einmal setzen, dann brauchst du nämlich nur eine handleClick Funktion. Außerdem, hast du absichtlich einen fünfseitigen Würfel gebaut? :D